pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/fst/FstIndexType.java

```diff
@@ -77,7 +78,32 @@ public Map<String, FstIndexConfig> fromIndexLoadingConfig(IndexLoadingConfig ind
     Set<String> fstIndexColumns = indexLoadingConfig.getFSTIndexColumns();
     for (String column : indexLoadingConfig.getAllKnownColumns()) {
       if (fstIndexColumns.contains(column)) {
-        FstIndexConfig conf = new FstIndexConfig(indexLoadingConfig.getFSTIndexType());
+        FSTType fstType = indexLoadingConfig.getFSTIndexType();
+
+        TableConfig tableConfig = indexLoadingConfig.getTableConfig();
```

Comment:
add a comment or wrap those changes to a helper method, as to get `fstType` from index props.